src/python/ensembl/io/genomio/database/meta_getter.py

```diff
@@ -111,7 +102,12 @@ def parse_args(arg_list: list[str] | None) -> argparse.Namespace:
 
 
 def main(arg_list: list[str] | None = None) -> None:
+    """Main script entry-point.
+
+    Args:
+        arg_list: Arguments to parse passing list to parse_args().
+    """
     args = parse_args(arg_list)
     init_logging_with_args(args)
 
-    _ = get_meta_values(server_url=args.url, db_name=args.database_name, input_queries=args.meta_keys_list)
+    _ = get_meta_values(server_url=args.url, db_name=args.database_name, meta_keys=args.meta_keys_list)
```

Comment:
`args.url` already contains the database in this case, so I would suggest changing the method's signature:
```suggestion
    _ = get_meta_values(db_url=args.url, meta_keys=args.meta_keys_list)
```
(although the code you have at the minute works, you are just overwriting the database in the URL for the same value inside =`get_meta_values`)